content/rwh.js

```diff
@@ -981,6 +976,31 @@ var ReplyWithHeader = {
     }
   },
 
+  DateFmt: function(d_str) {
+    var options = {
+        weekday: "short",
+        year: "numeric",
+        month: "short",
+        day: "numeric",
+    }
+    var time_opt = {
```

Comment:
@ofa- Please use `camelCase` for variables. May be something like `timeOpts`.

Question; is it beneficial to declare variable `options` & `timeOpts` outside the function?